### DISCUSSION TITLE
Free ProtonVPN configs

### DISCUSSION THREAD

**Discussion by Hiroshime** [haugene/vpn-configs-contrib#105] [Q&A] [answered]
Hello, i just registered for a free account on ProtonVPN but i could not find the free servers opvn config files here in this repository.
Is there a reason for the free servers not being here or it's just because it's not updated?
If it's just not updated can i help pulling those free server files?

Thanks.

**Comment by ilike2burnthing** [comment, marked as answer]
ProtonVPN's free tier does not support P2P, so there are none available here where only P2P servers have been added.
